src/navigation/wallet/screens/AddWallet.tsx

```diff
@@ -522,6 +570,62 @@ const AddWallet: React.FC<AddWalletScreenProps> = ({navigation, route}) => {
                 </RowContainer>
               </AdvancedOptions>
             )}
+
+            {showOptions && !singleAddressCurrency && (
+              <AdvancedOptions>
+                <RowContainer
+                  activeOpacity={1}
+                  onPress={() => {
+                    setSingleAddress(!singleAddress);
+                  }}>
+                  <Column>
+                    <OptionTitle>Single Address</OptionTitle>
+                  </Column>
+                  <CheckBoxContainer>
+                    <Checkbox
+                      checked={singleAddress}
+                      onPress={() => {
+                        setSingleAddress(!singleAddress);
+                      }}
+                    />
+                  </CheckBoxContainer>
+                </RowContainer>
+
+                {singleAddress && (
+                  <>
+                    <Info style={{marginHorizontal: 10}}>
+                      <InfoTriangle />
+
+                      <InfoHeader>
+                        <InfoImageContainer infoMargin={'0 8px 0 0'}>
+                          <InfoSvg />
+                        </InfoImageContainer>
+
+                        <InfoTitle>Single Address Wallet</InfoTitle>
+                      </InfoHeader>
+                      <InfoDescription>
+                        The single address feature will force the wallet to only
+                        use one address rather than generating new addresses.
+                      </InfoDescription>
+
+                      <VerticalPadding>
+                        <TouchableOpacity
+                          onPress={() => {
+                            Haptic('impactLight');
+                            dispatch(
+                              openUrlWithInAppBrowser(
+                                'https://support.bitpay.com/hc/en-us/articles/360015920572-Setting-up-the-Single-Address-Feature-for-your-BitPay-Wallet',
```

Comment:
nit: since this URL is used in more than one place(multisig, add wallet...), it can be add to contans/index.ts URL section.

Reply:
Thanks